sql/moz-fx-data-shared-prod/bugzilla_dev_metrics/dataset_metadata.yaml

```diff
@@ -0,0 +1,14 @@
+friendly_name: bugzilla_metrics
+dataset_base_acl: syndicate
+description: bugzilla_metrics syndicate views
+user_facing: true
+labels: {}
+syndication:
+  prod:
+    syndicated_project: "moz-fx-bugzilla-prod"
```

Comment:
Actually, reviewing https://github.com/mozilla-it/webservices-infra/blob/main/bugzilla/tf/dev/bigquery.tf it seems like this dataset is stored in your nonprod project:

```suggestion
    syndicated_project: "moz-fx-bugzilla-nonprod"
```